js/header.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable  no-unused-vars */
```

Comment:
Kindly don't disable the eslint rules. Please comply with the standard configurations requested by microverse